src/components/portrait/portrait-initials.component.js

```diff
@@ -44,7 +44,7 @@ const PortraitInitials = ({
 
     // Select a font family to support different language characters
     // like Arial
-    context.font = `${Math.round(canvas.width / 2.4)}px Lato, Arial`;
+    context.font = `${Math.round(canvas.width / 2.4)}px "Sage UI", Arial`;
```

Comment:
I wonder if this is the cause of the intermittent chromatic failures, when the Sage UI font has not loaded and we end up using Arial instead.

Reply:
If this is the case, perhaps it would be better to load the Sage UI font via `design-tokens` instead of the CDN?

Reply:
I think `rel="preload"` will give us the best _chance_ of it being loaded before the component is rendered, otherwise the font could be downloaded when the component first renders.